packages/assertions/src/expectations/index.ts

```diff
@@ -6,6 +6,7 @@ export * from './endsWith';
 export * from './equals';
 export * from './includes';
 export * from './isAfter';
+export * from './isTrue';
```

Comment:
Apologies for nitpicking, but could we maintain alphabetical order of exports, please?

Reply:
Fixed